src/components/Contacts/ContactDetails/ContactDetailsTab/ContactDetailsTab.tsx

```diff
@@ -224,7 +224,10 @@ export const ContactDetailsTab: React.FC<ContactDetailTabProps> = ({
               <ContactDetailLoadingPlaceHolder variant="rect" />
             </>
           ) : (
-            <ContactDetailsTabPeople data={data?.contact} />
+            <ContactDetailsTabPeople
+              data={data?.contact}
+              accountListId={accountListId}
```

Comment:
Do you want this as a prop or should it just use the `useAccountListId` hook in the child components? Maybe there's issues with it not being available on first render.

Reply:
Yeah, I tried using ```useAccountListId```and ran into that issue.